### PR TITLE
Long running migration docs

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 1]
```diff
@@ -40,8 +40,11 @@ For a Cortex XSOAR version lower than 8, the `Listen Port` parameter is required
...
-3. In a web browser, go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
+2. (For Cortex XSOAR 6.x only) In the **Server Configuration** section, verify that the `instance.execute.external.<instance_name>` key is set to `true`. If this key does not exist, click **+ Add Server Configuration** and add the `instance.execute.external.<instance_name>` and set the value to `true`. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
+3.  In a web browser, go to:
+   (For Cortex XSOAR 6.x) `https://*<xsoar_address>*/instance/execute/*<instance_name>*`
+      (For Cortex XSOAR 8) `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+     (For Cortex XSIAM) `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:47]:
```suggestion
      (For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`

```

File: Packs/EDL/Integrations/EDL/README.md [hunk 2]
```diff
@@ -144,8 +143,12 @@ In order to get the list of all available fields to search by, you can configure
...
-3. In a web browser, go to `https://*<demisto_address>*/instance/execute/*<instance_name>*` .
+2. (Cortex XSOAR 6.x only) In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
+3. In a web browser, go to:
+   (For Cortex XSOAR 6.x) `https://*<xsoar_address>*/instance/execute/*<instance_name>*`
+   (For Cortex XSOAR 8) `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+   (For Cortex XSIAM) `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/README.md:150]:
```suggestion
  (For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

File: Packs/WebFileRepository/Integrations/WebFileRepository/README.md [hunk 3]
```diff
@@ -37,12 +37,15 @@ In a web browser, go to **`http://<cortex-xsoar-server-address>:<listen_port>`**
...
-3. In a web browser, go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
+2. (For Cortex XSOAR 6.x) In the **Server Configuration** section, verify that the `instance.execute.external.<instance_name>` key is set to `true`. If this key does not exist, click **+ Add Server Configuration** and add the `instance.execute.external.<instance_name>` and set the value to `true`. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
+3. In a web browser, 
+   (For Cortex XSOAR 6.x) go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
 
    In Multi Tenant environments, go to `https://<cortex-xsoar-address>/acc_<account name>/instance/execute/<instance_name>/`
 
-
+   (For Cortex XSOAR 8) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+   (For Cortex XSIAM) go to `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/WebFileRepository/Integrations/WebFileRepository/README.md:47]:
```suggestion
   (For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 4]
```diff
@@ -20,9 +20,14 @@ You can add a collection description as is done in *collection1_name*, or enter
...
-Use one of the following options:
-- **https://*demisto_address*/instance/execute/*instance_name*/{taxii2_api_endpoint}/**
-- **http://*demisto_address*:*listen_port*/{taxii2_api_endpoint}/**
+(For Cortex XSOAR 6.x) Use one of the following options:
+
+- `https://<xsoar_address>/instance/execute/<instance_name>/<taxii2_api_endpoint>/`
+- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
+
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:29]:
```suggestion
(For Cortex XSOAR 8 or Cortex XSIAM):
- `https://ext-<tenant>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 5]
```diff
@@ -21,15 +21,30 @@ You can add a collection description as is doen in `collection1_name`, or enter
...
+- **https://*xsoar_address*/instance/execute/*instance_name/{taxii2_api_endpoint}/**
+- **http://*xsoar_address*:*listen_port/{taxii2_api_endpoint}/**
+
+(For Cortex XSOAR 8) Use one of the following options: 
+- `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
+- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
+
+(For Cortex XSIAM) Use one of the following options: 
+- `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
+- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:34]:
```suggestion
(For Cortex XSOAR 8 or Cortex XSIAM):
- `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
  When running on an engine:  `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 6]
```diff
@@ -152,10 +157,19 @@ TIM fields (system generated and custom). An example of these two related object
...
+  2. You can use the `!taxii-server-list-collections` command in order to get a list of your server's collections and their IDs. You can also do it manually by running `curl https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/threatintel/collections/ | jq .` to get a list of the collections available and on your TAXII server. From the list, copy the correct ID of the collection you want to ingest.
+  
+ **For Cortex XSIAM**
+  1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>/threatintel/`
+  2. You can use the `!taxii-server-list-collections` command in order to get a list of your server's collections and their IDs. You can also do it manually by running `curl https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>/threatintel/collections/ | jq .` to get a list of the collections available and on your TAXII server. From the list, copy the correct ID of the collection you want to ingest.
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:170]:
```suggestion
**For Cortex XSOAR 8 or Cortex XSIAM**
  1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/threatintel/`
  2. You can use the `!taxii-server-list-collections` command in order to get a list of your server's collections and their IDs. You can also do it manually by running `curl https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/threatintel/collections/ | jq .` to get a list of the collections available and on your TAXII server. From the list, copy the correct ID of the collection you want to ingest.
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 7]
```diff
@@ -21,15 +21,30 @@ You can add a collection description as is doen in `collection1_name`, or enter
...
 To access the TAXII service by instance name, make sure *Instance execute external* is enabled. 
 
 1. In Cortex XSOAR, go to **Settings > About > Troubleshooting**.
-2. In the **Server Configuration** section, verify that the *instance.execute.external* key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+2. (For Cortex XSOAR 6.x only) In the **Server Configuration** section, verify that the *instance.execute.external* key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+3. You can now trigger the webhook URL:
+
+   - (For Cortex XSOAR 6) In a web browser, go to https://*<xsoar_address>*/instance/execute/*<instance_name>* .
+   -  (For Cortex XSOAR 8) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+   - (For Cortex XSIAM) go to `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:46]:
```suggestion
   -  (For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/README.md [hunk 8]
```diff
@@ -14,16 +14,24 @@ The collections are defined by a JSON object in the following format:
...
-To view the available TAXII services, visit the discovery service in one of the following options:
-
-- `https://*demisto_address*/instance/execute/*instance_name/taxii-discovery-service`
+(For Cortex XSOAR 6.x)
+- `https://*demisto_address*/ins Use one of the following options:tance/execute/*instance_name/taxii-discovery-service`
 - `http://*demisto_address*:*listen_port*/taxii-discovery-service`
 
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/`
+- `http://demisto_address:listen_port/{taxii2_api_endpoint}/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/README.md:23]:
```suggestion
   For running on an engine: `http://demisto_address:listen_port/{taxii2_api_endpoint}/`
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/README.md [hunk 9]
```diff
@@ -14,16 +14,24 @@ The collections are defined by a JSON object in the following format:
...
 ## How to Access the TAXII Service
 
-To view the available TAXII services, visit the discovery service in one of the following options:
-
-- `https://*demisto_address*/instance/execute/*instance_name/taxii-discovery-service`
+(For Cortex XSOAR 6.x)
+- `https://*demisto_address*/ins Use one of the following options:tance/execute/*instance_name/taxii-discovery-service`
 - `http://*demisto_address*:*listen_port*/taxii-discovery-service`
 
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/README.md:21]:
```suggestion
(For Cortex XSOAR 8 or Cortex XSIAM):
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/README.md [hunk 10]
```diff
@@ -14,16 +14,24 @@ The collections are defined by a JSON object in the following format:
...
+- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/README.md:22]:
```suggestion
- `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/`
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/README.md [hunk 11]
```diff
@@ -14,16 +14,24 @@ The collections are defined by a JSON object in the following format:
...
 
 1. In Cortex XSOAR, go to **Settings > About > Troubleshooting**.
-2. In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+2. (For Cortex XSOAR 6.x) In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+3. You can now trigger the webhook URL:
+   - For Cortex XSOAR 6.x: `<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>`. For example, https://my.demisto.live/instance/execute/taxiiserver. Note that the string instance does not refer to the name of your XSOAR instance, but rather is part of the URL.
+   -  (For Cortex XSOAR 8) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+   - (For Cortex XSIAM) go to `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/README.md:34]:
```suggestion
   -  (For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md [hunk 12]
```diff
@@ -14,16 +14,31 @@ The collections are defined by a JSON object in the following format:
...
+
+- **https://*demisto_address*/instance/execute/*instance_name/taxii-discovery-service**
+- **http://*demisto_address*:*listen_port/taxii-discovery-service**
+
+(For Cortex XSOAR 8) Use one of the following options:
+https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>
+http://xsoar_address:listen_port/{taxii2_api_endpoint}/
+
+(For Cortex XSIAM) Use one of the following options:
+https://ext-/.crtx..paloaltonetworks.com/xsiam/instance/execute/
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md:27]:
```suggestion
(For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
  When running on an engine: http://xsoar_address:listen_port/{taxii2_api_endpoint}/
```

File: Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md [hunk 13]
```diff
@@ -14,16 +14,31 @@ The collections are defined by a JSON object in the following format:
...
+http://xsiam_address:listen_port/{taxii2_api_endpoint}/
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md:28]:
```suggestion

```

File: Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md [hunk 14]
```diff
@@ -14,16 +14,31 @@ The collections are defined by a JSON object in the following format:
...
 To access the TAXII service by instance name, make sure ***Instance execute external*** is enabled. 
 
 1. In Cortex XSOAR, go to **Settings > About > Troubleshooting**.
-2. In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+2. (For Cortex XSOAR 6.x) In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+3. You can now trigger the webhook URL:
+
+   - For Cortex XSOAR 6.x: **<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>**. For example, https://my.demisto.live/instance/execute/taxiiserver. Note that the string instance does not refer to the name of your XSOAR instance, but rather is part of the URL.
+For Cortex XSOAR 8: https://ext-/.crtx..paloaltonetworks.com/xsoar/instance/execute/
+For Cortex XSIAM: https://ext-/.crtx..paloaltonetworks.com/xsiam/instance/execute/
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md:41]:
```suggestion
(For Cortex XSOAR 8 or Cortex XSIAM) `https://ext-<tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

**Comment by ShahafBenYakir** [comment]
PR required force merge due to a known issue in taxii server tests

**Pull request merged**